### DISCUSSION TITLE
What is chadwm using to create the systray?

### DISCUSSION THREAD

**Discussion by heysokam** [siduck/chadwm#189] [Q&A] [answered]
I've been using chadwm for a while, but I'm trying to learn dwm a bit more.
I noticed that chadwm has a systray, but I don't see the systray patch listed in the readme :thinking:
What does chadwm use for achieving the systray behavior? Is it the systray patch (unlisted in readme), or is it something else?

**Comment by siduck** [comment, marked as answer]
@heysokam systray patch ofc, i forgot to add it in readme i think

**Reply by heysokam** [reply]
> @siduck
> @heysokam systray patch ofc, i forgot to add it in readme i think

I was just reading about `status2d`, and it has a note in its readme saying that it also bundles `systray` inside it.
That might be why you "forgot" to add it, since you probably applied the changes from that patch instead :thinking:
